Build is failing as of this morning: error: 'WLR_SCENE_NODE_BLUR' undeclared

## Info

mango version:
mango 0.10.5(release)
wlroots version: 0.19.2

## Crash track

```
> Building NixOS configuration
these 12 derivations will be built:
  /nix/store/45wbn2d7d39xrciqfixir4mvb83nhc0p-mango-nightly.drv
  /nix/store/iy58gbclh5dh70blw47mx072z2wdvz8g-greetd.toml.drv
  /nix/store/k24anvwkl62li7njpkfq29zx4m8qrihs-unit-greetd.service.drv
  /nix/store/vqrpjhvknrj4nfzb87ssnqlb9a1fakca-home-manager-path.drv
  /nix/store/pxdr2zr49l2sysnprf91fjqa3pcl34qc-hm_fontconfigconf.d10hmfonts.conf.drv
  /nix/store/cnrsqfgplf4b5jvy7s22q2qcv8vmx870-home-manager-files.drv
  /nix/store/7vrqsxkhkwdzw17yyxpxxabbqf46kk3i-home-manager-generation.drv
  /nix/store/mr39hr3rxadcq5zrm9f217xswhag2g7g-unit-home-manager-gregf.service.drv
  /nix/store/93gva9yn17hf6avpm62m1l1jmgq9k4ab-system-units.drv
  /nix/store/zhc8gs9wwaqyxaja66jp7jd7wnpxyns5-user-environment.drv
  /nix/store/6sxg5pz9dqlwybka1jbsywbhvjxifs1b-etc.drv
  /nix/store/kgxqrg9mhp3nly6bszw9y2089kc6yrn9-nixos-system-cromartie-25.11.20251105.ffcdcf9.drv
mango> building '/nix/store/45wbn2d7d39xrciqfixir4mvb83nhc0p-mango-nightly.drv'
mango> Running phase: unpackPhase
mango> unpacking source archive /nix/store/fjqclpr7abia0gfchznb9ji6n4rxdz08-source
mango> source root is source
mango> Running phase: patchPhase
mango> Running phase: updateAutotoolsGnuConfigScriptsPhase
mango> Running phase: configurePhase
mango> mesonConfigurePhase flags: --prefix=/nix/store/i3qaf16fnsrcnw946743j06vl27hn6rs-mango-nightly --libdir=/nix/store/i3qaf16fnsrcnw946743j06vl27hn6rs-mango-nightly/lib --libexecdir=/nix/store/i3qaf16fnsrcnw946743j06vl27hn6rs-mango-nightly/libexec --bindir=/nix/store/i3qaf16fnsrcnw946743j06vl27hn6rs-mango-nightly/bin --sbindir=/nix/store/i3qaf16fnsrcnw946743j06vl27hn6rs-mango-nightly/sbin --includedir=/nix/store/i3qaf16fnsrcnw946743j06vl27hn6rs-mango-nightly/include --mandir=/nix/store/i3qaf16fnsrcnw946743j06vl27hn6rs-mango-nightly/share/man --infodir=/nix/store/i3qaf16fnsrcnw946743j06vl27hn6rs-mango-nightly/share/info --localedir=/nix/store/i3qaf16fnsrcnw946743j06vl27hn6rs-mango-nightly/share/locale -Dauto_features=enabled -Dwrap_mode=nodownload --buildtype=plain
mango> The Meson build system
mango> Version: 1.9.1
mango> Source dir: /build/source
mango> Build dir: /build/source/build
mango> Build type: native build
mango> Project name: mango
mango> Project version: 0.10.5
mango> C compiler for the host machine: gcc (gcc 14.3.0 "gcc (GCC) 14.3.0")
mango> C linker for the host machine: gcc ld.bfd 2.44
mango> C++ compiler for the host machine: g++ (gcc 14.3.0 "g++ (GCC) 14.3.0")
mango> C++ linker for the host machine: g++ ld.bfd 2.44
mango> Host machine cpu family: x86_64
mango> Host machine cpu: x86_64
mango> Program wayland-scanner found: YES (/nix/store/pd0fbk9yrygb8c5p36j4cazpf6lw8v75-wayland-scanner-1.24.0-bin/bin/wayland-scanner)
mango> Found pkg-config: YES (/nix/store/0skggkamysbh0ksc3582lb69wzrpabcg-pkg-config-wrapper-0.29.2/bin/pkg-config) 0.29.2
mango> Run-time dependency wayland-protocols found: YES 1.45
mango> protocols/meson.build:3: WARNING: Project does not target a minimum version but uses feature deprecated since '0.56.0': dependency.get_pkgconfig_variable. use dependency.get_variable(pkgconfig : ...) instead
mango> Library m found: YES
mango> Run-time dependency xcb found: YES 1.17.0
mango> Run-time dependency xcb-icccm found: YES 0.4.2
mango> Run-time dependency wayland-server found: YES 1.24.0
mango> Run-time dependency wlroots-0.19 found: YES 0.19.2
mango> Run-time dependency xkbcommon found: YES 1.11.0
mango> Run-time dependency libinput found: YES 1.29.1
mango> Run-time dependency wayland-client found: YES 1.24.0
mango> Run-time dependency libpcre2-8 found: YES 10.46
mango> Run-time dependency scenefx-0.4 found: YES 0.4.1
mango> Program git found: NO
mango> Build targets in project: 2
mango> WARNING: Deprecated features used:
mango>  * 0.56.0: {'dependency.get_pkgconfig_variable'}
mango>
mango> mango 0.10.5
mango>
mango>   User defined options
mango>     auto_features: enabled
mango>     bindir       : /nix/store/i3qaf16fnsrcnw946743j06vl27hn6rs-mango-nightly/bin
mango>     buildtype    : plain
mango>     includedir   : /nix/store/i3qaf16fnsrcnw946743j06vl27hn6rs-mango-nightly/include
mango>     infodir      : /nix/store/i3qaf16fnsrcnw946743j06vl27hn6rs-mango-nightly/share/info
mango>     libdir       : /nix/store/i3qaf16fnsrcnw946743j06vl27hn6rs-mango-nightly/lib
mango>     libexecdir   : /nix/store/i3qaf16fnsrcnw946743j06vl27hn6rs-mango-nightly/libexec
mango>     localedir    : /nix/store/i3qaf16fnsrcnw946743j06vl27hn6rs-mango-nightly/share/locale
mango>     mandir       : /nix/store/i3qaf16fnsrcnw946743j06vl27hn6rs-mango-nightly/share/man
mango>     prefix       : /nix/store/i3qaf16fnsrcnw946743j06vl27hn6rs-mango-nightly
mango>     sbindir      : /nix/store/i3qaf16fnsrcnw946743j06vl27hn6rs-mango-nightly/sbin
mango>     wrap_mode    : nodownload
mango>
mango> Found ninja-1.13.1 at /nix/store/asj4vdlgiqpcqc45lq29r4hv4wlfbl63-ninja-1.13.1/bin/ninja
mango> mesonConfigurePhase: enabled\ parallel\ building
mango> Running phase: buildPhase
mango> build flags: -j8
mango> [1/48] Generating 'mango.p/cursor-shape-v1-protocol.c'
mango> [2/48] Generating 'mango.p/pointer-constraints-unstable-v1-protocol.c'
mango> [3/48] Generating 'mango.p/ext-image-capture-source-v1-protocol.c'
mango> [4/48] Generating 'mango.p/xdg-shell-protocol.c'
mango> [5/48] Generating 'mango.p/ext-image-copy-capture-v1-protocol.c'
mango> [6/48] Generating 'mango.p/ext-foreign-toplevel-list-v1-protocol.c'
mango> [7/48] Generating 'mango.p/tablet-v2-protocol.c'
mango> [8/48] Generating 'mango.p/ext-workspace-v1-protocol.c'
mango> [9/48] Generating 'mango.p/tearing-control-v1-protocol.c'
mango> [10/48] Generating 'mango.p/wlr-foreign-toplevel-management-unstable-v1-protocol.c'
mango> [11/48] Generating 'mango.p/wlr-layer-shell-unstable-v1-protocol.c'
mango> [12/48] Generating 'mango.p/dwl-ipc-unstable-v2-protocol.c'
mango> [13/48] Generating 'mango.p/wlr-output-power-management-unstable-v1-protocol.c'
mango> [14/48] Generating 'mango.p/pointer-constraints-unstable-v1-protocol.h'
mango> [15/48] Generating 'mango.p/cursor-shape-v1-protocol.h'
mango> [16/48] Generating 'mango.p/xdg-shell-protocol.h'
mango> [17/48] Generating 'mango.p/ext-image-copy-capture-v1-protocol.h'
mango> [18/48] Generating 'mango.p/ext-image-capture-source-v1-protocol.h'
mango> [19/48] Generating 'mango.p/ext-foreign-toplevel-list-v1-protocol.h'
mango> [20/48] Generating 'mango.p/tablet-v2-protocol.h'
mango> [21/48] Generating 'mango.p/tearing-control-v1-protocol.h'
mango> [22/48] Generating 'mango.p/wlr-foreign-toplevel-management-unstable-v1-protocol.h'
mango> [23/48] Generating 'mango.p/ext-workspace-v1-protocol.h'
mango> [24/48] Generating 'mango.p/dwl-ipc-unstable-v2-protocol.h'
mango> [25/48] Generating 'mango.p/wlr-output-power-management-unstable-v1-protocol.h'
mango> [26/48] Generating 'mango.p/wlr-layer-shell-unstable-v1-protocol.h'
mango> [27/48] Generating 'mmsg.p/dwl-ipc-unstable-v2-protocol.c'
mango> [28/48] Generating 'mmsg.p/dwl-ipc-unstable-v2-protocol.h'
mango> [29/48] Compiling C object mango.p/meson-generated_ext-image-copy-capture-v1-protocol.c.o
mango> [30/48] Compiling C object mango.p/meson-generated_ext-image-capture-source-v1-protocol.c.o
mango> [31/48] Compiling C object mango.p/meson-generated_ext-foreign-toplevel-list-v1-protocol.c.o
mango> [32/48] Compiling C object mango.p/meson-generated_pointer-constraints-unstable-v1-protocol.c.o
mango> [33/48] Compiling C object mango.p/meson-generated_cursor-shape-v1-protocol.c.o
mango> [34/48] Compiling C object mango.p/meson-generated_ext-workspace-v1-protocol.c.o
mango> [35/48] Compiling C object mango.p/meson-generated_tablet-v2-protocol.c.o
mango> [36/48] Compiling C object mango.p/meson-generated_xdg-shell-protocol.c.o
mango> [37/48] Compiling C object mango.p/meson-generated_wlr-foreign-toplevel-management-unstable-v1-protocol.c.o
mango> [38/48] Compiling C object mango.p/meson-generated_tearing-control-v1-protocol.c.o
mango> [39/48] Compiling C object mango.p/meson-generated_wlr-output-power-management-unstable-v1-protocol.c.o
mango> [40/48] Compiling C object mango.p/meson-generated_wlr-layer-shell-unstable-v1-protocol.c.o
mango> [41/48] Compiling C object mango.p/meson-generated_dwl-ipc-unstable-v2-protocol.c.o
mango> [42/48] Compiling C object mmsg.p/meson-generated_dwl-ipc-unstable-v2-protocol.c.o
mango> [43/48] Compiling C object mango.p/src_common_util.c.o
mango> [44/48] Compiling C object mango.p/src_ext-protocol_wlr_ext_workspace_v1.c.o
mango> [45/48] Compiling C object mmsg.p/mmsg_mmsg.c.o
mango> [46/48] Linking target mmsg
mango> [47/48] Compiling C object mango.p/src_mango.c.o
mango> FAILED: [code=1] mango.p/src_mango.c.o
mango> gcc -Imango.p -I. -I.. -I/nix/store/27czksni7m3kji791lvfdl40n99f71q8-libxcb-1.17.0-dev/include -I/nix/store/lm7zhswyrln1krx9rs0nsix4g17dwkx1-libxcb-wm-0.4.2-dev/include -I/nix/store/jpf3jx348gh8nvxwxmq31z9ak2hwv81s-scenefx-0.4.1-git/include/scenefx-0.4 -I/nix/store/465lj8xn3vp4xfand16iqlwx1kqgc4id-wayland-1.24.0-dev/include -I/nix/store/ambdp6ax8s1kxvsqgi30fxh7bwrxhdxw-wlroots-0.19.2/include/wlroots-0.19 -I/nix/store/anzq0yqha04vzs47v32nisyqf9dqak9j-libxkbcommon-1.11.0-dev/include -I/nix/store/9s4nkf07fcl0rbc4d36lda3b877cjc9w-libinput-1.29.1-dev/include -I/nix/store/7f7jxszkvdpwp3y6py3mfhmm8nsh72zl-pcre2-10.46-dev/include -fdiagnostics-color=always -D_FILE_OFFSET_BITS=64 -Wall -Winvalid-pch -g -Wno-unused-function -DWLR_USE_UNSTABLE -D_POSIX_C_SOURCE=200809L '-DVERSION="0.10.5(release)"' '-DSYSCONFDIR="/etc"' -DXWAYLAND -MD -MQ mango.p/src_mango.c.o -MF mango.p/src_mango.c.o.d -o mango.p/src_mango.c.o -c ../src/mango.c
mango> In file included from ../src/mango.c:878:
mango> ../src/animation/client.h: In function 'buffer_set_effect':
mango> ../src/animation/client.h:249:17: error: implicit declaration of function 'wlr_scene_blur_set_size'; did you mean 'wlr_scene_rect_set_size'? [-Wimplicit-function-declaration]
mango>   249 |                 wlr_scene_blur_set_size(c->blur, c->animation.current.width - 2 * c->bw,
mango>       |                 ^~~~~~~~~~~~~~~~~~~~~~~
mango>       |                 wlr_scene_rect_set_size
mango> ../src/animation/client.h:252:17: error: implicit declaration of function 'wlr_scene_blur_set_corner_radius'; did you mean 'wlr_scene_buffer_set_corner_radius'? [-Wimplicit-function-declaration]
mango>   252 |                 wlr_scene_blur_set_corner_radius(c->blur, border_radius,
mango>       |                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
mango>       |                 wlr_scene_buffer_set_corner_radius
mango> In file included from ../src/mango.c:879:
mango> ../src/animation/common.h: In function 'scene_node_snapshot':
mango> ../src/animation/common.h:195:14: error: 'WLR_SCENE_NODE_BLUR' undeclared (first use in this function); did you mean 'WLR_SCENE_NODE_BUFFER'?
mango>   195 |         case WLR_SCENE_NODE_BLUR:
mango>       |              ^~~~~~~~~~~~~~~~~~~
mango>       |              WLR_SCENE_NODE_BUFFER
mango> ../src/animation/common.h:195:14: note: each undeclared identifier is reported only once for each function it appears in
mango> ../src/mango.c: In function 'iter_layer_scene_buffers':
mango> ../src/mango.c:2090:44: error: invalid use of undefined type 'struct wlr_scene_blur'
mango>  2090 |         wlr_scene_node_set_enabled(&l->blur->node, true);
mango>       |                                            ^~
mango> ../src/mango.c:2091:9: error: implicit declaration of function 'wlr_scene_blur_set_transparency_mask_source' [-Wimplicit-function-declaration]
mango>  2091 |         wlr_scene_blur_set_transparency_mask_source(l->blur, buffer);
mango>       |         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
mango> ../src/mango.c:2095:17: error: implicit declaration of function 'wlr_scene_blur_set_should_only_blur_bottom_layer' [-Wimplicit-function-declaration]
mango>  2095 |                 wlr_scene_blur_set_should_only_blur_bottom_layer(l->blur, true);
mango>       |                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
mango> ../src/mango.c: In function 'maplayersurfacenotify':
mango> ../src/mango.c:2151:27: error: implicit declaration of function 'wlr_scene_blur_create'; did you mean 'wlr_scene_buffer_create'? [-Wimplicit-function-declaration]
mango>  2151 |                 l->blur = wlr_scene_blur_create(l->scene, 0, 0);
mango>       |                           ^~~~~~~~~~~~~~~~~~~~~
mango>       |                           wlr_scene_buffer_create
mango> ../src/mango.c:2151:25: error: assignment to 'struct wlr_scene_blur *' from 'int' makes pointer from integer without a cast [-Wint-conversion]
mango>  2151 |                 l->blur = wlr_scene_blur_create(l->scene, 0, 0);
mango>       |                         ^
mango> ../src/mango.c:2152:56: error: invalid use of undefined type 'struct wlr_scene_blur'
mango>  2152 |                 wlr_scene_node_lower_to_bottom(&l->blur->node);
mango>       |                                                        ^~
mango> ../src/mango.c: In function 'iter_xdg_scene_buffers':
mango> ../src/mango.c:3537:52: error: invalid use of undefined type 'struct wlr_scene_blur'
mango>  3537 |                 wlr_scene_node_set_enabled(&c->blur->node, true);
mango>       |                                                    ^~
mango> ../src/mango.c:3545:52: error: invalid use of undefined type 'struct wlr_scene_blur'
mango>  3545 |                 wlr_scene_node_set_enabled(&c->blur->node, false);
mango>       |                                                    ^~
mango> ../src/mango.c: In function 'mapnotify':
mango> ../src/mango.c:3658:17: error: assignment to 'struct wlr_scene_blur *' from 'int' makes pointer from integer without a cast [-Wint-conversion]
mango>  3658 |         c->blur = wlr_scene_blur_create(c->scene_surface, 0, 0);
mango>       |                 ^
mango> ../src/mango.c:3659:48: error: invalid use of undefined type 'struct wlr_scene_blur'
mango>  3659 |         wlr_scene_node_lower_to_bottom(&c->blur->node);
mango>       |                                                ^~
mango> ninja: build stopped: subcommand failed.
error: Cannot build '/nix/store/45wbn2d7d39xrciqfixir4mvb83nhc0p-mango-nightly.drv'.
       Reason: builder failed with exit code 1.
       Output paths:
         /nix/store/i3qaf16fnsrcnw946743j06vl27hn6rs-mango-nightly
       Last 25 log lines:
       > ../src/mango.c:2151:27: error: implicit declaration of function 'wlr_scene_blur_create'; did you mean 'wlr_scene_buffer_create'? [-Wimplicit-function-declaration]
       >  2151 |                 l->blur = wlr_scene_blur_create(l->scene, 0, 0);
       >       |                           ^~~~~~~~~~~~~~~~~~~~~
       >       |                           wlr_scene_buffer_create
       > ../src/mango.c:2151:25: error: assignment to 'struct wlr_scene_blur *' from 'int' makes pointer from integer without a cast [-Wint-conversion]
       >  2151 |                 l->blur = wlr_scene_blur_create(l->scene, 0, 0);
       >       |                         ^
       > ../src/mango.c:2152:56: error: invalid use of undefined type 'struct wlr_scene_blur'
       >  2152 |                 wlr_scene_node_lower_to_bottom(&l->blur->node);
       >       |                                                        ^~
       > ../src/mango.c: In function 'iter_xdg_scene_buffers':
       > ../src/mango.c:3537:52: error: invalid use of undefined type 'struct wlr_scene_blur'
       >  3537 |                 wlr_scene_node_set_enabled(&c->blur->node, true);
       >       |                                                    ^~
       > ../src/mango.c:3545:52: error: invalid use of undefined type 'struct wlr_scene_blur'
       >  3545 |                 wlr_scene_node_set_enabled(&c->blur->node, false);
       >       |                                                    ^~
       > ../src/mango.c: In function 'mapnotify':
       > ../src/mango.c:3658:17: error: assignment to 'struct wlr_scene_blur *' from 'int' makes pointer from integer without a cast [-Wint-conversion]
       >  3658 |         c->blur = wlr_scene_blur_create(c->scene_surface, 0, 0);
       >       |                 ^
       > ../src/mango.c:3659:48: error: invalid use of undefined type 'struct wlr_scene_blur'
       >  3659 |         wlr_scene_node_lower_to_bottom(&c->blur->node);
       >       |                                                ^~
       > ninja: build stopped: subcommand failed.
       For full logs, run:
         nix log /nix/store/45wbn2d7d39xrciqfixir4mvb83nhc0p-mango-nightly.drv
error: Cannot build '/nix/store/iy58gbclh5dh70blw47mx072z2wdvz8g-greetd.toml.drv'.
       Reason: 1 dependency failed.
       Output paths:
         /nix/store/zg8drzxq4xpfz7vijm4frh8q5f8xzb2k-greetd.toml
error: Cannot build '/nix/store/vqrpjhvknrj4nfzb87ssnqlb9a1fakca-home-manager-path.drv'.
       Reason: 1 dependency failed.
       Output paths:
         /nix/store/237gll77076nv6n9g2zsa56b7d3vlnfq-home-manager-path
error: Cannot build '/nix/store/7vrqsxkhkwdzw17yyxpxxabbqf46kk3i-home-manager-generation.drv'.
       Reason: 1 dependency failed.
       Output paths:
         /nix/store/9qv4y9mdjwd468ivknlkp5g79n42j6lh-home-manager-generation
error: Cannot build '/nix/store/k24anvwkl62li7njpkfq29zx4m8qrihs-unit-greetd.service.drv'.
       Reason: 1 dependency failed.
       Output paths:
         /nix/store/fnkpipyhdi2c6x87wbz1601gspmka6za-unit-greetd.service
error: Cannot build '/nix/store/zhc8gs9wwaqyxaja66jp7jd7wnpxyns5-user-environment.drv'.
       Reason: 1 dependency failed.
       Output paths:
         /nix/store/gys31fz5s8yl20nkkymlvblvqlin99xz-user-environment
error: Cannot build '/nix/store/6sxg5pz9dqlwybka1jbsywbhvjxifs1b-etc.drv'.
       Reason: 1 dependency failed.
       Output paths:
         /nix/store/nbc4w2f227jgnhksncg8s2zd06ycfxyp-etc
error: Cannot build '/nix/store/kgxqrg9mhp3nly6bszw9y2089kc6yrn9-nixos-system-cromartie-25.11.20251105.ffcdcf9.drv'.
       Reason: 1 dependency failed.
       Output paths:
         /nix/store/qgc63vrm0vmpc4b58phv3yyyqcb5c1q7-nixos-system-cromartie-25.11.20251105.ffcdcf9
┏━ 8 Errors:
 ⋮
┃        Reason: 1 dependency failed.
┃        Output paths:
┃          /nix/store/zg8drzxq4xpfz7vijm4frh8q5f8xzb2k-greetd.toml
┃ error: Cannot build '/nix/store/45wbn2d7d39xrciqfixir4mvb83nhc0p-mango-nightly.drv'.
┃        Reason: builder failed with exit code 1.
┃        Output paths:
┃          /nix/store/i3qaf16fnsrcnw946743j06vl27hn6rs-mango-nightly
┃        Last 25 log lines:
┃        > ../src/mango.c:2151:27: error: implicit declaration of function 'wlr_scene_blur_create'; did you mean 'wlr_scene_buffer_create'? [-Wimplicit-function-declaration]
┃        >  2151 |                 l->blur = wlr_scene_blur_create(l->scene, 0, 0);
┃        >       |                           ^~~~~~~~~~~~~~~~~~~~~
┃        >       |                           wlr_scene_buffer_create
┃        > ../src/mango.c:2151:25: error: assignment to 'struct wlr_scene_blur *' from 'int' makes pointer from integer without a cast [-Wint-conversion]
┃        >  2151 |                 l->blur = wlr_scene_blur_create(l->scene, 0, 0);
┃        >       |                         ^
┃        > ../src/mango.c:2152:56: error: invalid use of undefined type 'struct wlr_scene_blur'
┃        >  2152 |                 wlr_scene_node_lower_to_bottom(&l->blur->node);
┃        >       |                                                        ^~
┃        > ../src/mango.c: In function 'iter_xdg_scene_buffers':
┃        > ../src/mango.c:3537:52: error: invalid use of undefined type 'struct wlr_scene_blur'
┃        >  3537 |                 wlr_scene_node_set_enabled(&c->blur->node, true);
┃        >       |                                                    ^~
┃        > ../src/mango.c:3545:52: error: invalid use of undefined type 'struct wlr_scene_blur'
┃        >  3545 |                 wlr_scene_node_set_enabled(&c->blur->node, false);
┃        >       |                                                    ^~
┃        > ../src/mango.c: In function 'mapnotify':
┃        > ../src/mango.c:3658:17: error: assignment to 'struct wlr_scene_blur *' from 'int' makes pointer from integer without a cast [-Wint-conversion]
┃        >  3658 |         c->blur = wlr_scene_blur_create(c->scene_surface, 0, 0);
┃        >       |                 ^
┃        > ../src/mango.c:3659:48: error: invalid use of undefined type 'struct wlr_scene_blur'
┃        >  3659 |         wlr_scene_node_lower_to_bottom(&c->blur->node);
┃        >       |                                                ^~
┃        > ninja: build stopped: subcommand failed.
┃        For full logs, run:
┃          nix log /nix/store/45wbn2d7d39xrciqfixir4mvb83nhc0p-mango-nightly.drv
┣━ Dependency Graph:
┃    ┌─ ⏸ user-environment waiting for 1 ⏵
┃    │              ┌─ ⏸ home-manager-path waiting for 1 ⏵
┃    │           ┌─ ⏸ hm_fontconfigconf.d10hmfonts.conf
┃    │        ┌─ ⏸ home-manager-files
┃    │     ┌─ ⏸ home-manager-generation
┃    │  ┌─ ⏸ unit-home-manager-gregf.service
┃    │  │     ┌─ ⏵ mango-nightly (buildPhase) ⏱ 2s
┃    │  │  ┌─ ⏸ greetd.toml
┃    │  ├─ ⏸ unit-greetd.service
┃    ├─ ⏸ system-units
┃ ┌─ ⏸ etc
┃ ⏸ nixos-system-cromartie-25.11.20251105.ffcdcf9
┣━━━ Builds
┗━ ∑ ⏵ 1 │ ✔ 0 │ ⏸ 11 │ ⚠ Exited with 8 errors reported by nix at 12:09:19 after 25s
Error:
   0: Failed to build configuration
   1: Command exited with status Exited(1)

Location:
   src/commands.rs:693
error: Recipe `rebuild` failed on line 13 with exit code 1
```

```nix
$ nix-info -m
 - system: `"x86_64-linux"`
 - host os: `Linux 6.17.6-zen1, NixOS, 25.11 (Xantusia), 25.11.20251105.ffcdcf9`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.31.2`
 ```

--- a/flake.nix
+++ b/flake.nix
@@ -25,12 +25,13 @@
 
       perSystem = {
         config,
+        system,
         pkgs,
         ...
       }: let
-        inherit (pkgs) callPackage ;
+        inherit (pkgs) callPackage;
         mango = callPackage ./nix {
-          inherit (inputs.scenefx.packages.${pkgs.stdenv.hostPlatform.system}) scenefx;
+          scenefx = inputs.scenefx.packages.${system}.default;
         };
         shellOverride = old: {
           nativeBuildInputs = old.nativeBuildInputs ++ [];

